guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Target Groups
+
+The general assumption is that the audience is familiar with the following terms and does not require the explanation of technical concepts behind them:
+
+- Kubernetes
+- Docker and containers
+
+<!-- Information about generic target groups is based on the SAP Styleguide: https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/546dfb06e80c4005aabc4795d548fe35.html -->
+
+### Decision maker
+
+**Tasks**: Assess the software to make sure that it will meet the company's needs. Needs the facts – not just marketing spin – before signing on the dotted line.
+**Goal**: Purchase the right solution for the company, and ensure stakeholders back this decision.
+
+### Software developer
+
+**Tasks**: Interested in technical topics. Solid knowledge of programming languages. Experienced in programming and development projects. Expert in the technical or business area. Uses and contributes to community content.
+**Goal**: Develop and enhance software.
+
+### Admin/Operations
+
+**Tasks**: Deals with installation, upgrades, system troubleshooting.
+**Goal**: Support the ongoing operations and evolution of the Kyma implementation.
+
+## Documentation Structure
+
+On the Kyma website, we have five main tabs containing multiple documents each, plus a glossary.
+
+### Overview
+
+Target Group: Decision Makers (Tech Leads) and newbies.
+Contains a quick overview of the idea behind Kyma, a diagram of the main areas with a brief explanation, and, as needed, longer documents going into the details of each main area.
+
+### Getting Started
+
+Target Group: Software Developers who quickly want to see what they can do with Kyma.
+Contains a guide/tutorial that covers typical steps you need to get started.
+
+### Deep Dive/Tutorials
+
+Target Group: Software Developers leveraging all the Kyma functionalities.
+Under this tab, there are subtabs according to main areas (except UI – Busola and CLI are mentioned as needed within the instructions of the respective main area). Documents in the subtabs contain "how-to" instructions that enable users to accomplish a task
+
+### Operations
+
+Target Group: Admins/Operators who make sure the Kyma cluster is configured as needed and keeps running in a healthy and secure way.
+Contains installation and configuration instructions, backup info, troubleshooting guides…
+
+### Technical References
+
+Target Group: Users who want need to look up specific detailed information.
+Contains the architecture diagrams, configuration charts, etc,; no explanation of concepts or instructions.
+
+### Glossary
+
+Target Group: Anyone who wants to look up terms they’re not familiar with.
+Explains basic terms, with a focus on terms specific to Kyma.
```

Comment:
Or 
```suggestion
Explains basic Kyma-related terms.
```
or
```Explains basic Kyma and Kyma-related terms.```
or
```Explains basic Kyma-specific and Kyma-related terms.```

Reply:
Wasn't sure we are allowed to hyphenate "Kyma", thus the longer phrase.